presto-native-execution/presto_cpp/main/tests/JsonSignatureParserTest.cpp

```diff
@@ -197,5 +197,52 @@ TEST_F(JsonSignatureParserTest, multiple) {
   EXPECT_EQ(signature1->argumentTypes()[1].baseName(), "varchar");
 }
 
+TEST_F(JsonSignatureParserTest, dynamic) {
+  auto input = R"(
```

Comment:
Please add a function which uses a complex type for either paramType or outputType